apps/frontend/web/src/components/CharacterCard.tsx

```diff
@@ -57,15 +57,15 @@ function CharacterCard({
       if (jsonResponse) {
         toast({
           title: `Hire ${gameCharacter.character.name}`,
-          description: `Congratulations, you hire : ${gameCharacter.character.name}!`,
+          description: `Congratulations, you hired : ${gameCharacter.character.name}!`,
```

Comment:
```suggestion
          description: `Congratulations, you hired: ${gameCharacter.character.name}!`,
```
attention l'espace :eyes: